readme.md

```diff
@@ -36,38 +36,14 @@ That's it. Skip to [Getting started](#getting-started).
 
 ### Manually
 
-1. Either…
-  - Clone this repo
-  - add it as a submodule, or
-  - just download [`pure.zsh`](pure.zsh) and [`async.zsh`](async.zsh)
+1. Clone this repo into /some/dir
```

Comment:
Maybe we could use a more descriptive folder name, like `$HOME/pure`? It might be nicer if the path is usable *out of the box*, so to speak.